frontend/src/app/components/EventCard.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+
+import * as React from 'react';
+import { Event } from '../api/calendarClient';
+import IconButton from '@mui/material/IconButton';
+import DeleteIcon from '@mui/icons-material/Delete';
+import CreateIcon from '@mui/icons-material/Create';
+
+const EventCard = (props: { event: Event }) => {
+  return (
+    <div style={{
+      color: 'white',
+      backgroundColor: '#E9967A',
+      borderRadius: 4,
+      fontSize: '1rem',
+      marginBottom: '8px'
+    }}>            
+        {props.event.startAt.toString().split('T')[1]} -
```

Comment:
You could use a date formatter here instead of splitting strings in the template. [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) is a reliable solution, if you don't want to add another dependency. If you're open to using a date utility library, you can check its `format` function.